benchmark/src/main/java/org/eclipse/rdf4j/benchmark/FastRdfsForwardChainingSailBenchmark.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+
+
+package org.eclipse.rdf4j.benchmark;
+
+import org.eclipse.rdf4j.repository.sail.SailRepository;
+import org.eclipse.rdf4j.sail.inferencer.fc.FastRdfsForwardChainingSail;
+import org.eclipse.rdf4j.sail.memory.MemoryStore;
+
+/**
+ * @author Håvard Mikkelsen Ottestad
+ */
+public class FastRdfsForwardChainingSailBenchmark extends RdfsBenchmark {
+
+
```

Comment:
The formatting settings turns this

```java
    @Benchmark
    @BenchmarkMode(Mode.AverageTime)
    @OutputTimeUnit(TimeUnit.MILLISECONDS)
    public void simple() throws IOException {
```

into this

```java
	@Benchmark @BenchmarkMode(Mode.AverageTime) @OutputTimeUnit(TimeUnit.MILLISECONDS) public void simple()
			throws IOException
	{
```

@ansell before I commit this, is this what the code should look like, or did Intellij mess up something in the import?